llvm/test/Bitcode/amdgcn-make.buffer.rsrc.ll

```diff
@@ -0,0 +1,12 @@
+; RUN: llvm-as < %s | llvm-dis | FileCheck %s
```

Comment:
auto generate check lines pls.

Reply:
I don't think auto-upgrade tests are ever autogenerated? Unless I've just been seeing old examples.

Reply:
aha I c. just out of curiosity, what happens if you simply do `opt -S %s -o | FileCheck %s`?

Reply:
What I've done before is generate the check with opt, and then switch the run line back to use llvm-as + llvm-dis